core/src/main/java/com/linecorp/armeria/common/Cookies.java

```diff
@@ -13,32 +13,52 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
-package com.linecorp.armeria.server.annotation;
+package com.linecorp.armeria.common;
 
 import static java.util.Objects.requireNonNull;
 
 import java.util.Set;
 
 import com.google.common.collect.ImmutableSet;
 
-import io.netty.handler.codec.http.cookie.Cookie;
-
 /**
- * An interface which holds decoded {@link Cookie} instances for an HTTP request.
+ * An immutable {@link Set} of {@link Cookie}s.
  */
 public interface Cookies extends Set<Cookie> {
 
+    /**
+     * Returns an empty {@link Set} if {@link Cookie}s.
+     */
+    static Cookies empty() {
+        return DefaultCookies.EMPTY;
+    }
+
     /**
      * Creates an instance with a copy of the specified set of {@link Cookie}s.
```

Comment:
I think either this could be `array of {@link Cookie}s` for consistency with the other javadoc (this parameter isn't a set), or even better is probably to use `specified set of {@link Cookie}s` in the other javadoc same as here since semantically they're the same method.

Reply:
Changed to 'specified set of Cookies'